charts/spire/templates/tests/test-connection.yaml

```diff
@@ -1,15 +0,0 @@
-apiVersion: v1
```

Comment:
Why did you delete this file?

Reply:
WGET doesn't work on a grpc service. This caused the CI to fail as it will also run the tests. We would have to create a new test-connection deployment based on a grpc_cli to test the grpc endpoints. For that we need a trustworthy docker image. Considering building one for this purpose.

Reply:
Clear, thanks for clarifying.